devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -244,13 +244,16 @@ def resetActiveSound() {
 		sendHubCommand(onOffCmd(0x00))
 	}
 	sendEvent([name: "alarm", value: "off"])
-	sendEvent([name: "chime", value: "off"])
 }
 
 def setActiveSound(soundId) {
 	String childDni = "${device.deviceNetworkId}:${soundId}"
 	def child = childDevices.find { it.deviceNetworkId == childDni }
-	child?.sendEvent([name: "chime", value: "chime"])
+	if(soundId >= 3 || soundId <= 5) {
+		child?.sendEvent(name: "chime", value: "on", displayed: false)
+	} else {
+		child?.sendEvent(name: "alarm", value: "both", displayed: false)
```

Comment:
unless you're sending both a "charm" and an "alarm" event for every device I think you'll still see the network error.

Reply:
Yeah, you're going to need to send both events, or do one of my suggestions in the original creation and have two different child device types; one for doorbell and one for siren. That might be hard to do now that the device has already been created. Though you *might* be able to re-type child devices (keep in mind if anyone has managed to create rules based on those, they would likely break).